hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanPlanner.java

```diff
@@ -400,6 +416,16 @@ private String getLatestVersionBeforeCommit(List<FileSlice> fileSliceList, Hoodi
     return deletePaths;
   }
 
+  public List<CleanFileInfo> getDeletePartitionPaths(String partitionPath) {
```

Comment:
so, in case of DeletePartition, we just include the partition path (directory) to CleanFileInfo and do not add explicit file groups/files within partition is it?